include/deal.II/lac/arpack_solver.h

```diff
@@ -76,7 +76,16 @@ extern "C" void dneupd_(int *rvec, char *howmany, int *select, double *d,
  * <code>dnaupd</code> work and also how to set the parameters appropriately
  * please take a look into the ARPACK manual.
  *
- * @author Baerbel Janssen, Agnieszka Miedlar, 2010.
+ * @note Whenever you eliminate degrees of freedom using
+ * ConstraintMatrix, you generate spurious eigenvalues and
+ * eigenvectors. If you make sure that the diagonals of eliminated
+ * matrix rows are all equal to one, you get a single additional
+ * eigenvalue. But beware that some functions in deal.II set these
+ * diagonals to rather arbitrary (from the point of view of
+ * eigenvalue problems) values. See also @ref step_36 "step-36" for an
```

Comment:
If you just say step-36 (without the @ref and quotes) then our scripts expand this to the text you have.